tipsy/Constants.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Constants.swift
+//  tipsy
+//
+//  Created by Lia Zadoyan on 8/1/17.
+//  Copyright © 2017 Lia Zadoyan. All rights reserved.
+//
+
+struct UserDefaultKeys {
```

Comment:
I like adding a `Constants` class to all my projects for defining strings, etc that may be reused. Less chance of messing up a misspelling with a string constant.